src/modules/weather/weather.module.js

```diff
@@ -0,0 +1,199 @@
+                    },
+                ));
+
+        this.container.append(nextDaysDiv);
+    }
+
+
+    _createWeatherElementIn(parent, firstElement, icon, maxTemperature, minTemperature, isHorizontal = false) {
+
+        const weatherElement = isHorizontal
+            ? createRow()
+            : document.createElement('div');
+
+        weatherElement.append(
+            createFirstElementDiv(firstElement),
+            createWeatherIconDiv(icon),
+            createTemperatureRangeDiv(maxTemperature, minTemperature),
+        );
+
+        parent.append(weatherElement);
+
+        function createTemperatureRangeDiv(maxTemperature) {
+            const temperatureRangeDiv = isHorizontal
+                ? createColumn()
+                : createRow();
+
+            const maxTempDiv = createTempDiv(maxTemperature);
+            maxTempDiv.classList.add('fw-bold');
+
+            temperatureRangeDiv.append(
+                maxTempDiv,
+                createTempDiv(minTemperature),
+            );
+
+            return temperatureRangeDiv;
+
+            function createTempDiv(temperature) {
+                const temperatureDiv = isHorizontal
+                    ? document.createElement('div')
+                    : createColumn();
+                temperatureDiv.innerText = `${temperature}°C`;
+                return temperatureDiv;
+            }
+        }
+
+        function createFirstElementDiv(value) {
+            const firstElementDiv = isHorizontal
+                ? createColumn()
+                : document.createElement('div');
+
+            switch (value) {
+                case 0:
+                    firstElementDiv.innerText = 'Sun';
+                    break;
+                case 1:
+                    firstElementDiv.innerText = 'Mon';
+                    break;
+                case 2:
+                    firstElementDiv.innerText = 'Tue';
+                    break;
+                case 3:
+                    firstElementDiv.innerText = 'Wed';
+                    break;
+                case 4:
+                    firstElementDiv.innerText = 'Thu';
+                    break;
+                case 5:
+                    firstElementDiv.innerText = 'Fri';
+                    break;
+                case 6:
+                    firstElementDiv.innerText = 'Sat';
+                    break;
+                default:
+                    firstElementDiv.innerText = value;
+                    break;
+            }
+
+            return firstElementDiv;
+        }
+
+        function createWeatherIconDiv(icon) {
+            const weatherIconDiv = isHorizontal
+                ? createColumn()
+                : document.createElement('div');
+            const img = document.createElement('img');
+            img.src = `./content/img/weather/${icon}-s.png`;
+            weatherIconDiv.appendChild(img);
+            return weatherIconDiv;
+        }
+
+    }
+}
+
+
+function createRow() {
```

Comment:
you can create files under /src/utils called 'createRow.js' and 'createColumn.js' and import it from there. probably we will use it in other components